File: 2024-Semester-1/PB (VP)/Seminar Exercises/Exercise Week 6 - 31.10.2024/ex-1.py
num = int(input("Enter a number: "))

if num % 3 == 0 and num % 5 == 0:
    print("Числото е кратно на 3 и 5")
elif num % 3 != 0 and num % 5 != 0:
    print("Числото не е кратно на 3 и 5")
elif num % 3 == 0:
    print("Числото е кратно на 3")
elif num % 5 == 0:
    print("Числото е кратно на 5")

File: 2025-Semester-2/BPL (BPE)/Lab Exercises/Exercise Week 8 - 24.03.2025/ex-1.c
//
// Created by Kristiyan Petsanov on 24.03.25.
//

#include <stdio.h>

int main() {
  int n;

  printf("Enter the number of elements in the two dimensional array: ");
  scanf("%d", &n);

  int arr[n][n];

  printf("Enter the elements of the array: ");
  for (int i = 0; i < n; i++) {
    printf("Enter the elements of row %d: ", i + 1);
    printf("\n");
    for (int j = 0; j < n; j++) {
      printf("Enter the element of column %d: ", j + 1);
      scanf("%d", &arr[i][j]);
    }
  }

  printf("The elements of the main diagonal are: ");
  for (int i = 0; i < n; i++) {
    printf("%d ", arr[i][i]);
  }
  printf("\n");

  printf("The elements of the secondary diagonal are: ");
  for (int i = 0; i < n; i++) {
    printf("%d ", arr[i][n - i - 1]);
  }
  printf("\n");

  printf("The elements above the main diagonal are: ");
  for (int i = 0; i < n; i++) {
    for (int j = i + 1; j < n; j++) {
      printf("%d ", arr[i][j]);
    }
  }
  printf("\n");

  printf("The elements below the main diagonal are: ");
  for (int i = 1; i < n; i++) {
    for (int j = 0; j < i; j++) {
      printf("%d ", arr[i][j]);
    }
  }

  return 0;
}